recipes/maeparser/test.cpp

```diff
@@ -0,0 +1,6 @@
+#include <maeparser/MaeBlock.hpp>
+#include <maeparser/MaeConstants.hpp>
+
+int main() {
+    auto block = std::make_shared<schrodinger::mae::Block>(schrodinger::mae::CT_BLOCK);
+}
```

Comment:
As a general note: please add a single trailing newline to all files (git standard).